packages/lib-classifier/src/store/TutorialStore.js

```diff
@@ -122,6 +122,7 @@ const TutorialStore = types
           // We're not setting the store state to error because
           // we do not want to prevent the tutorial from rendering
           console.error(error)
+          self.setMediaResources([])
```

Comment:
`attachedMedia` is a map and I'd rather leave it as undefined rather than set it to an empty array if there is no attached media for the tutorial

Reply:
@roger pointed out that this generator doesn't yield anything on error, but I'm not sure what it should yield, if anything.